src/shared/theme/theme.ts

```diff
@@ -0,0 +1,19 @@
+export const theme = {
+  shadow: '0px 1px 6px rgba(0, 0, 0, 0.25)',
+  colors: {
+    primary: '#F78710',
+    secondary: { dark: '#0863E3', light: '#468BEB' },
+    text: {
+      body: '#292825',
+      heading: {
+        light: '#FFFFFF',
+        dark: '#16408F',
+      },
+    },
+    background: {
+      light: '#FFF8EE',
+      sand: '#FFE6C5',
+      gradient: 'linear-gradient(270deg, #FF901A 0%, #FB6203 100%)',
+    },
+  },
```

Comment:
@Wendlereis could you help me with the organization of those colors? I tried to separate by the semantics but I've been using some colors in more than one context and I don't know if it hindered more than it helped

Reply:
This granular approach is great! I'll go with it